docs/content/gameobject/api-workerconnector.md

```diff
@@ -46,23 +47,23 @@ Returns: the name of the selected cloud deployment.
 ```csharp
 protected virtual ReceptionistConfig GetReceptionistConfig(string workerType)
 ```
-This method is only used when connecting through the [Receptionist connection flow](../glossary.md#receptionist-connection-flow).
+This method is only used when connecting through the [Receptionist connection flow]({{urlRoot}}/content/glossary#receptionist-connection-flow).
 
 Parameters:
-* `string workerType`: The [type of the worker](../glossary.md#worker-types)
+* `string workerType`: The [type of the worker]({{urlRoot}}/content/glossary#worker-types)
 
-Returns: a connection configuration to connect using the [Receptionist connection flow](../glossary.md#receptionist-connection-flow).
+Returns: a connection configuration to connect using the [Receptionist connection flow]({{urlRoot}}/content/glossary#receptionist-connection-flow).
 
 ```csharp
 protected virtual LocatorConfig GetLocatorConfig(string workerType)
 ```
-This method is only used when connecting through the [Locator connection flow](../glossary.md#locator-connection-flow).
+This method is only used when connecting through the [Locator connection flow]({{urlRoot}}/content/glossary#locator-connection-flow).
 
 Parameters:
-  * `string workerType`: The [type of the worker](../glossary.md#worker-types)
+  * `string workerType`: The [type of the worker]({{urlRoot}}/content/glossary#worker-types)
 
-Returns: a connection configuration to connect using the [Locator connection flow](../glossary.md#locator-connection-flow) stored
-as a [`LocatorConfig`](../../../tree/master/workers/unity/Packages/com.improbable.gdk.core/Config/LocatorConfig.cs) object.
+Returns: a connection configuration to connect using the [Locator connection flow]({{urlRoot}}/content/glossary#locator-connection-flow) stored
+as a [`LocatorConfig`](https://github.com/spatialos/gdk-for-unity/workers/unity/Packages/com.improbable.gdk.core/Config/LocatorConfig.cs) object.
```

Comment:
This might need to be `https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.core/Config/LocatorConfig.cs`. If I open the current link, I'm getting a "Not found" error.

Reply:
Thanks, done.